tests/test_C2_economic_functions.py

```diff
@@ -23,7 +23,7 @@
     "fuel_price": 1.3,
     "fuel_price_change_annual": 0,
     "project_lifetime": project_life,
-    "wacc": wacc,
+    "discount_factor": discount_factor,
```

Comment:
please do the same here, ie. replace "discount_factor" = DISCOUNTFACTOR and "project_duration" = PROJECT_DURATION in this file.

Reply:
@smartie2076 do you mean "project_lifetime" ?

Reply:
yes